cmd/helm/repo.go

```diff
@@ -107,16 +107,20 @@ func runRepoRemove(cmd *cobra.Command, args []string) error {
 }
 
 func runRepoIndex(cmd *cobra.Command, args []string) error {
-	if err := checkArgsLength(2, len(args), "path to a directory", "url of chart repository"); err != nil {
+	if err := checkArgsLength(1, len(args), "path to a directory"); err != nil {
 		return err
 	}
 
 	path, err := filepath.Abs(args[0])
 	if err != nil {
 		return err
 	}
+	url := ""
+	if len(args) == 2 {
```

Comment:
Yes, didn't realise it was limited to 1 argument, above in checkArgs function.

I still want to allow the "repo_url" to be provided, but want to make it optional..
Updating the PR. now...